actions/run.py

```diff
@@ -13,12 +13,12 @@
 class SlackAction(Action):
 
     def run(self, **kwargs):
-        if kwargs.get('token', None) is None:
-            kwargs['token'] = self.config['action_token']
-
         return self._do_request(kwargs)
 
-    def _do_request(self, params):
+    def _do_request(self, params, files=None):
```

Comment:
Should be able to remove the current `run` method, rename `_do_request` to `run`, and tweak the arguments a bit:

```python
    def run(self, files=None, **params):
```

I don't think `_do_request` is called anywhere else.